File: lab8/fromClassRepository/parrot.cpp
// Bob Roos
// Demonstration program for lab 8. This program is used in
// conjunction with the following files:
//       lab8demo.cpp
//       pirate.h
//       pirate.cpp
//       parrot.h

#include <string>
#include "parrot.h"

using namespace std;

Parrot::Parrot(string n) { Parrot::name = n; }

string Parrot::getName() { return Parrot::name; }

string Parrot::speak() { return "Awwwk!"; }

File: lab8/fromClassRepository/parrot.cs
// Bob Roos
// Lab 8 demonstration program
//
// See files "parrot.cs" and "lab8demo.cs" for the
// other parts of this program.

using System;
namespace ParrotNS
{
   // A Parrot has a name and always says "Awwwk!"
   class Parrot {
      private string name;
   
      // Constructor:
      public Parrot(string n) {
         name = n;
      }
   
      public string getName() { return name; }
   
      public string speak() { return "Awwwk!"; }
   }
}
